Stats/Supporting Files/el.lproj/Localizable.strings

```diff
@@ -2,7 +2,7 @@
 //  Localizable.strings
 //  Stats
 //
-//  Created by Serhiy Mytrovtsiy on 27/08/2020.
+//  Created by sudoxcess, Updated by vaionicle on 17/04/2023
```

Comment:
do not change the license. You can only add information about contributors.

Reply:
@exelban

Ok, Thanks for the info, I will update it and push it back